test/configCases/parsing/optional-chaining/index.js

```diff
@@ -0,0 +1,10 @@
+it("should correctly render defined data #1", () => {
+	expect(_VALUE_?._DEFINED_).toBe(1);
+});
+
+it("should correctly render defined data #2", () => {
+	const val1 = _VALUE_?._PROP_?._DEFINED_;
+	const val2 = _VALUE_?._PROP_?._UNDEFINED_;
```

Comment:
Should I keep this test case? Same problem
> Test is passing, but since we render "" ?? false need to find a way to load this tests in Node.js < 14. Maybe use babel before load or move this to configCases and handle there with babel.